### PR TITLE
Added basic test cases to donut chart

### PR DESCRIPTION
**Pull request by v-baambati**
Added basic test cases to donut chart and horizontalbar chart for all variants.
To run the test cases please use "npx playwright test" command also we need to run storybook before running the test cases.

### CODE REVIEW

File: packages/web-components/src/donut-chart/donut-chart.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { expect, fixtureURL } from '../helpers.tests.js';
+test.describe('Donut-chart - Basic', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(fixtureURL('components-donutchart--basic'));
+
+    await page.waitForFunction(() => customElements.whenDefined('fluent-donut-chart'));
+  });
+
+  test('Should render chart properly', async ({ page }) => {
+    const element = page.locator('fluent-donut-chart');
```
[the remark this answers is not in the document]
Comment (v-baambati) [reply packages/web-components/src/donut-chart/donut-chart.spec.ts:11]: For now added test cases on existing example which is there in the stories.

[the remark this answers is not in the document]
Comment (AtishayMsft) [reply packages/web-components/src/donut-chart/donut-chart.spec.ts:11]: Create new examples for testing. Dont use existing story examples.

File: packages/web-components/src/donut-chart/donut-chart.spec.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    await expect(calloutRoot).toHaveCSS('opacity', '1');
+    const calloutLegendText = await element.locator('.calloutLegendText');
+    await expect(calloutLegendText).toHaveText('first');
+    const calloutContentY = await element.locator('.calloutContentY');
+    await expect(calloutContentY).toHaveText('20000');
+    const secondPath = element.getByLabel('second,');
+    await secondPath.dispatchEvent('mouseover');
+    await expect(calloutRoot).toHaveCSS('opacity', '1');
+    await expect(calloutLegendText).toHaveText('second');
+    await expect(calloutContentY).toHaveText('39000');
```
Comment (AtishayMsft) [inline packages/web-components/src/donut-chart/donut-chart.spec.ts:109]: could you add tests for dark mode and RTL

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.spec.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+import { expect, fixtureURL } from '../helpers.tests.js';
+
+test.describe('horizontalbarchart - Basic', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(fixtureURL('components-horizontalbarchart--basic'));
+    await page.waitForFunction(() => customElements.whenDefined('fluent-horizontalbarchart'));
+  });
+
+  test('Should render horizontalbarchart properly', async ({ page }) => {
+    const element = page.locator('fluent-horizontalbarchart');
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.spec.ts:12]: horizontal bar chart has 3 variants. Add individual tests for all variants separately.

File: packages/web-components/src/donut-chart/donut-chart.spec.ts [hunk 4]
```diff
@@ -0,0 +1,162 @@
+import { test } from '@playwright/test';
+import { expect, fixtureURL } from '../helpers.tests.js';
+import { teamsDarkTheme } from '@fluentui/tokens';
+import { colorNeutralBackground1 } from '../theme/design-tokens.js';
+
+test.describe('Donut-chart - Basic', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(fixtureURL('components-donutchart--basic'));
+    await page.setContent(/* html */ `
+      <fluent-donut-chart value-inside-donut="39,000" inner-radius="55" data="{&quot;chartTitle&quot;:&quot;Donut chart basic example&quot;,&quot;chartData&quot;:[{&quot;legend&quot;:&quot;first&quot;,&quot;data&quot;:20000},{&quot;legend&quot;:&quot;second&quot;,&quot;data&quot;:39000}]}">
```
Comment (AtishayMsft) [inline packages/web-components/src/donut-chart/donut-chart.spec.ts:10]: use json stringify to represent the data

**Pull request merged**